content/en/real_user_monitoring/browser/monitoring_page_performance.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Monitoring Page Performance
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/real_user_monitoring/explorer/"
+    tag: "Documentation"
+    text: "Explore your views within Datadog"
+  - link: "/real_user_monitoring/explorer/analytics/"
+    tag: "Documentation"
+    text: "Build analytics upon your events"
+  - link: "/real_user_monitoring/dashboards/"
+    tag: "Documentation"
+    text: "RUM Dashboards"
+---
+
+When a user visits a page on your website, a new **RUM view event** is created. The view contains live page load and performance metrics. Performance metrics are sent to Datadog as the page finishes loading, but might be empty for a few seconds before they appear in the Datadog UI.
+
+## Identifying performance bottlenecks
+
+To find what's causing a page to under-perform:
+
+1. Set up and configure [RUM Browser monitoring][1] for your application.
+2. Open the [Performance Overview dashboard][2]. It provides you with a high level view of your monitored pages, and you can apply filters to find pages with performance problems.
```

Comment:
"From the RUM Applications page, head to the Performance Overview dashboard. [...]. Apply filters to narrow your search and uncover performance problems." or something like that.